.travis.yml

```diff
@@ -211,7 +211,8 @@ script:
   - SKIP_FILES1=".github|codeship-*|buddy.yml|appveyor.yml|circleci|distelli-manifest.yml"
   - SKIP_FILES2="|fast-forward-merge.sh|LICENSE|LICENSE.apache20|README.md|release.sh|RIGHTS.antlr"
   - SKIP_FILES3="|shippable.yml|shippable.sh|wercker.yml|wercker.sh|intellij-idea-inspections.xml"
-  - SKIP_FILES=$SKIP_FILES1$SKIP_FILES2$SKIP_FILES3
+  - SKIP_FILES4="|Jenkinsfile"
+  - SKIP_FILES=$SKIP_FILES1$SKIP_FILES2$SKIP_FILES3$SKIP_FILES4
```

Comment:
please add it to all CIs, you missed:
https://github.com/checkstyle/checkstyle/blob/master/.circleci/config.yml#L21  (pitest)
https://github.com/checkstyle/checkstyle/blob/master/distelli-manifest.yml#L9  (javadoc regression)

Just FYI we also have : buddy.yml, codeship-steps.yml , but such CIs are limited for `master` only , so I did not have much time to update them too to skip builds, so for now ok to skip them in update.

Reply:
Done, thanks!